app/frontend/courses/students/components/CoursesStudents__StudentOverlay.res

```diff
@@ -597,6 +577,56 @@ let make = (~studentId, ~userId) => {
               : React.null}
             {coachInfo(studentDetails)}
             {otherTeamMembers(setState, studentId, studentDetails)}
+            <div className="mt-4">
+              <div className="justify-between mt-8 flex space-x-2">
+                <p className="text-sm font-semibold"> {"Milestone targets"->str} </p>
```

Comment:
@mahesh-krishnakumar Shouldn't Milestone targets be translated?

Reply:
Yes @shilendras

Reply:
@mahesh-krishnakumar @shilendras 

It will fixed when this is merged https://github.com/pupilfirst/pupilfirst/pull/1413/files#diff-eed799d5644932cd99f227be3756039c0a54868a45fd2048e3163531baaf235a